### PR TITLE
[ADD] 8.0 - Added purchase order report in qweb with notes

### PR DESCRIPTION
**Pull request by SerpentCS**
**Migrated Purchase Order Report in Qweb with Notes**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
The name has no sense now that is in QWeb. You should rename it to _purchase_order_report_with_note_ (singular as the guideline says).

File: purchase_order_webkit_with_notes/README.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
+
+=====================================
+Purchase Order Qweb Report With Notes	
+=====================================
+
+* This module replaces the webkit Quotation / Purchase Order report by
+  a qweb report and adds header and footer notes on the purchase order 
+  and quotation
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/README.rst]: add a period

File: purchase_order_webkit_with_notes/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+
+Installation
+============
+
+To install this module there is no external library required.
+
+1. Nothing required at all.
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/README.rst]: remove this line

File: purchase_order_webkit_with_notes/README.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+1. No Configuration required at all.
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/README.rst]: remove previous line and the enumeration here

File: purchase_order_webkit_with_notes/README.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+Usage
+=====
+
+* This module will be helpful to have the notes in header and footer for any Purchase Order.
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/README.rst]: This should be in introduction not in usage.
Indicate here where you can find the report or add a screenshot

File: purchase_order_webkit_with_notes/README.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+   :target: https://runbot.odoo-community.org/runbot/141/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/purchase-reporting/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/README.rst]: remove this part following newest README template

File: purchase_order_webkit_with_notes/README.rst [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Camptocamp
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/README.rst]: Remove or add a name+email

File: purchase_order_webkit_with_notes/README.rst [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+* Eficent Business and IT Consulting Services S.L. <contact@eficent.com>
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
+* Odoo Community Association (OCA)
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/README.rst]: remove

File: purchase_order_webkit_with_notes/__init__.py [hunk 8]
```diff
@@ -27,5 +27,4 @@
 #
 ##############################################################################
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/__init__.py]: remove this header (optional in init.py)

File: purchase_order_webkit_with_notes/__openerp__.py [hunk 9]
```diff
@@ -28,23 +28,22 @@
 ##############################################################################
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/__openerp__.py]: modify all headers following short header style of the OCA

File: purchase_order_webkit_with_notes/__openerp__.py [hunk 10]
```diff
@@ -28,23 +28,22 @@
...
-             'view/purchase_view.xml'],
-    'test': [],
+    'name': 'Purchase Order Qweb Report with notes',
+    'version': '8.0.1',
+    'category': 'Reports/Qweb',
+    "author": "Camptocamp,"
+              "Eficent Business and IT Consulting Services S.L., "
+              "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
+    'website': "http://www.camptocamp.com,"
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/__openerp__.py]: one website AFAIK

File: purchase_order_webkit_with_notes/__openerp__.py [hunk 11]
```diff
@@ -28,23 +28,22 @@
...
-for header and footer
-    """,
-    'author': "Camptocamp,Odoo Community Association (OCA)",
-    'website': 'http://www.camptocamp.com',
-    'depends': ['base', 'report_webkit', 'base_headers_webkit', 'purchase'],
-    'data': ['purchase_report.xml',
-             'view/purchase_view.xml'],
-    'test': [],
+    'name': 'Purchase Order Qweb Report with notes',
+    'version': '8.0.1',
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/__openerp__.py]: 8.0.1.0.0

File: purchase_order_webkit_with_notes/models/purchase_order.py [hunk 12]
```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline purchase_order_webkit_with_notes/models/purchase_order.py]: simplify all copyright headers

**Comment by pedrobaeza** [comment]
Please check errors in Runbot/Travis

File: purchase_order_report_with_note/README.rst [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+* This module replaces the webkit Quotation / Purchase Order report by
+  a qweb report and adds header and footer notes on the purchase order 
+  and quotation
+
+
+Installation
+============
+
+No specific installation requirements.
```
Comment (LoisRForgeFlow) [inline purchase_order_report_with_note/README.rst:17]: Remove empty section.

File: purchase_order_report_with_note/README.rst [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+
+
+Configuration
+=============
+
+No configuration is required.
```
Comment (LoisRForgeFlow) [inline purchase_order_report_with_note/README.rst:23]: Remove empty section.

File: purchase_order_webkit/__openerp__.py [hunk 15]
```diff
@@ -43,6 +43,6 @@
     'data': ['purchase_report.xml',
              'view/purchase_view.xml'],
     'test': [],
-    'installable': True,
+    'installable': False,
```
Comment (LoisRForgeFlow) [inline purchase_order_webkit/__openerp__.py:46]: Why not to remove it totally? Some need to keep it there? (just wondering)

**Comment by pedrobaeza** [comment]
Closing this as very old and without response.

**Pull request closed without merging**